src/ldp/http/metadata/LinkParser.ts

```diff
@@ -1,17 +1,27 @@
+import type { NamedNode } from '@rdfjs/types';
 import { DataFactory } from 'n3';
 import { getLoggerFor } from '../../../logging/LogUtil';
 import type { HttpRequest } from '../../../server/HttpRequest';
 import { parseParameters, splitAndClean, transformQuotedStrings } from '../../../util/HeaderUtil';
-import { RDF } from '../../../util/Vocabularies';
 import type { RepresentationMetadata } from '../../representation/RepresentationMetadata';
 import { MetadataParser } from './MetadataParser';
 
 /**
- * Parses Link headers with "rel=type" parameters and adds them as RDF.type metadata.
+ * Parses Link headers and adds them as metadata with the given predicate.
```

Comment:
```suggestion
 * Parses Link headers with a specific `rel` value and adds them as metadata with the given predicate.
```
I just realized this doens't parse all link headers but only those with a specific `rel` value, so we should still indicate this.